pkg/reconciler/v1alpha1/taskrun/taskrun_test.go

```diff
@@ -1219,6 +1231,28 @@ func TestUpdateStatusFromPod(t *testing.T) {
 		},
 	}} {
 		t.Run(c.desc, func(t *testing.T) {
+			observer, _ := observer.New(zap.InfoLevel)
+			logger := zap.New(observer).Sugar()
+			fakeClient := fakeclientset.NewSimpleClientset()
+			sharedInfomer := informers.NewSharedInformerFactory(fakeClient, 0)
+			pipelineResourceInformer := sharedInfomer.Tekton().V1alpha1().PipelineResources()
+			resourceLister := pipelineResourceInformer.Lister()
+			fakekubeclient := fakekubeclientset.NewSimpleClientset()
```

Comment:
this might be another place where the test setup could be a bit less complicated if you passed in your own functions instead of instantiating all the observers, listers, etc.